CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Bug Fixes:
  * Updated SWPC/NOAA links due to their new website.
  * Exposed the raw AIA color tables in `sunpy.cm.color_tables`.
  * Fixes `map` compatibility with Astropy 1.0.x.
+ * Corrected GOES satellites' operational time ranges so all data back to 1980 can be read into GOESLightCurve object
```

Comment:
yes every pr should have a changelog (unless it is a vanishingly stupid change).

Either pypi or github releases (https://github.com/sunpy/sunpy/releases) show the latest version. I should create the new header upon release.